damnit/gui/table.py

```diff
@@ -400,6 +405,14 @@ def headerData(self, col, orientation, role=Qt.ItemDataRole.DisplayRole):
         ):
             name = self._data.columns[col]
             return self._main_window.column_title(name)
+        elif(
+            orientation == Qt.Orientation.Vertical
+            and role == Qt.ItemDataRole.DisplayRole
+        ):
+            row = self._data.iloc[col]['Run']
+            if pd.isna(row):
+                row = ''
+            return row
```

Comment:
Is this a bugfix for something?

Reply:
This is what's making the 'frozen column' - it's actually the vertical (per-row) headers. The `headerData()` method is called both for column & row headers.

Reply:
Whoops thanks, I didn't read properly and thought this was in `.data()` :facepalm: